cmd/lakefs/cmd/run.go

```diff
@@ -132,7 +134,18 @@ var runCmd = &cobra.Command{
 		}
 		defer func() { _ = c.Close() }()
 
-		multipartsTracker := multiparts.NewTracker(dbPool)
+		var multipartsTracker multiparts.Tracker
+
+		if dbParams.KVEnabled {
+			kvStore, err := kv.Open(ctx, dbParams.Type, dbParams.ConnectionString)
+			if err != nil {
+				logger.WithError(err).Fatal("failed to open KV store")
+			}
+			defer kvStore.Close()
+			multipartsTracker = multiparts.NewTracker(kv.StoreMessage{Store: kvStore})
```

Comment:
I know `migration` isn't part of this PR, but I would add 
```suggestion
			// TODO: migration required
			kvStore, err := kv.Open(ctx, dbParams.Type, dbParams.ConnectionString)
			if err != nil {
				logger.WithError(err).Fatal("failed to open KV store")
			}
			defer kvStore.Close()
			multipartsTracker = multiparts.NewTracker(kv.StoreMessage{Store: kvStore})
```

Reply:
I'm not sure this will be the place where migration will happen, and since we don't know who might take this task, this TODO can easily be forgotten and left in the code. I'm confident we won't forget to implement the migration even without this TODO :)